fetch.bs

```diff
@@ -917,6 +919,14 @@ not always relevant and might require different behavior.
    <td>"<code>manifest</code>"
    <td><code>manifest-src</code>
    <td>HTML's <code>&lt;link rel=manifest></code>
+  <tr>
+   <td>"<code>prefetch</code>"
+   <td rowspan=2>""
+   <td rowspan=2><code>prefetch-src</code>
+   <td>HTML's <code>&lt;link rel=prefetch></code>
+  <tr>
+   <td>"<code>prerender</code>"
```

Comment:
Reading through https://github.com/w3c/webappsec-csp/commit/91adc4ab7beb21bcc05e3ebeeb035d2745c7817d, prefetch-src should also be listed for prerender?

Reply:
It is (that's what the "rowspan=2" is all about)

Reply:
Ah, right. Nevermind me, all good!